apps/web/src/services/asset-inventory/stores/metric-explorer-l-s-b-store.ts

```diff
@@ -0,0 +1,146 @@
+import { computed, reactive } from 'vue';
+
+import { defineStore } from 'pinia';
+
+import { SpaceConnector } from '@cloudforet/core-lib/space-connector';
+import { getCancellableFetcher } from '@cloudforet/core-lib/space-connector/cancellable-fetcher';
+
+import type { ListResponse } from '@/schema/_common/api-verbs/list';
+import { RESOURCE_GROUP } from '@/schema/_common/constant';
+import type { NamespaceGroupListParameters } from '@/schema/inventory-v2/namespace-group/api-verbs/list';
+import type { NamespaceGroupModel } from '@/schema/inventory-v2/namespace-group/model';
+import type { NamespaceListParameters } from '@/schema/inventory-v2/namespace/api-verbs/list';
+import type { NamespaceModel } from '@/schema/inventory-v2/namespace/model';
+import type { MetricListParameters } from '@/schema/inventory/metric/api-verbs/list';
+import type { MetricModel } from '@/schema/inventory/metric/model';
+
+import { useAppContextStore } from '@/store/app-context/app-context-store';
+
+interface MetricExplorerLSBPageStoreState {
+    groupLoading: boolean;
+    namespaceLoadingMap: Record<string, boolean>;
+    metricLoading: boolean;
+    namespaceGroupList: NamespaceGroupModel[];
+    namespaceMap: Record<string, NamespaceModel[]>
+    metricList: MetricModel[];
+    selectedNamespace?: NamespaceModel;
+}
+
+export const useMetricExplorerLSBPageStore = defineStore('metric-explorer-l-s-b', () => {
+    const appContextStore = useAppContextStore();
+
+    const _state = reactive({
+        isAdminMode: computed<boolean>(() => appContextStore.getters.isAdminMode),
+    });
```

Comment:
👀: Since the data is coming from the store's getters, how about naming it '_getters'? It's a minor change, but that's how it's been used in other stores!